### PR TITLE
provide factory method for object creation in test cases

### PR DESCRIPTION
**Pull request by Luolingwei**
1 create factory method for object creation in test cases
2 refactor existing code in test cases
3 improve score for ```pylint```

### CODE REVIEW

File: tests/suppliers_factory.py [hunk 1]
```diff
@@ -1,3 +1,26 @@
+
 """
 Test Factory to make fake objects for testing
-"""
\ No newline at end of file
+"""
+import factory
```
Comment (rymaspic) [inline tests/suppliers_factory.py:5]: Meet this error when running "nosetests"

`from .suppliers_factory import SupplierFactory
  File "/vagrant/tests/suppliers_factory.py", line 5, in <module>
    import factory
ModuleNotFoundError: No module named 'factory'`

I ran the "vagrant up" and "vagrant provision" commands and it seems it collected the factory-boy package. Is this the right way to reset the environment after you add new packaged requirements?

Comment (Luolingwei) [reply tests/suppliers_factory.py:5]: When you run vagrant up, all python packages defined in requirement.txt will NOT be reinstalled. It's only done in the VM provision process. So when we update our python package suit, you need to run `pip install -r requirements.txt` to install the new package.

**Review by rymaspic — COMMENTED** [review]
Reviewed, I think it can be merged.

**Comment by kunal15** [comment]
Looks good! Merging.

**Review by kunal15 — APPROVED** [review]

**Pull request merged**